app/server.py

```diff
@@ -95,30 +114,30 @@ async def get_appraisal(
     log_level = query_dict.get("log_level", "INFO")
     LOGGER.setLevel(LOGGER.setLevel(logging._nameToLevel[log_level]))
     message = query_dict["message"]
+    qid = str(uuid4())[:8]
     if "result" not in message:
         return JSONResponse(content={"status": "Rejected",
                                      "description": "No Results.",
                                      "job_id": qid}, status_code=200)
     callback = query_dict["callback"]
-    qid = str(uuid4())[:8]
     background_tasks.add_task(appraise, qid, message, callback)
     return JSONResponse(content={"status": "Accepted",
                                  "description": f"Appraising answers. Will send response to {callback}",
                                  "job_id": qid}, status_code=200)
 
 @APP.post("/sync_get_appraisal", response_model=Response)
 async def sync_get_appraisal(
-    query: Query
+    query: Query = Body(..., example=EXAMPLE)
 ):
     query_dict = query.dict()
-    log_level = query_dict.get("log_level", "INFO")
-    LOGGER.setLevel(LOGGER.setLevel(logging._nameToLevel[log_level]))
+    log_level = query_dict["log_level"] or "INFO"
```

Comment:
I believe this will crash if log level is not set. You still need the `.get()` in case it's not defined.